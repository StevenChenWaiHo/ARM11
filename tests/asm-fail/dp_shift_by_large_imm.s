sub r1, r2, r3, lsl #FFF
