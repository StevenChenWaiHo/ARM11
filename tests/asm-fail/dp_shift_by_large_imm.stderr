tests/asm-fail/dp_shift_by_large_imm.s:1:25: Invalid number #FFF
